tests/distinct/tnil.nim

```diff
@@ -0,0 +1,41 @@
+discard """
+  file: "tnil.nim"
+  output: "0x1\n\nnil\n\nnil\n\n"
```

Comment:
You can also have triple single quote string literals here and avoid the ``\n``'s. Just sayin', nothing you need to change.